recipe/meta.yaml

```diff
@@ -1,21 +1,18 @@
 {% set name = "transformers" %}
-{% set version = "4.29.2" %}
+{% set version = "4.32.0" %}
```

Comment:
4.32.1 seems to be already available.

Reply:
Yeah, there are valuable patches. I'll update it to 4.32.1.